### ISSUE TITLE
Error code 500 when trying to retrieve vectors for observation

### ISSUE THREAD

**Issue by boriskovar-m2ms** [m2ms/fragalysis-frontend#1165]
When calling /api/vector/ I get http code 500. From the last conversation with @kaliif it should work because this endpoint actually calculate vectors on the fly.

@phraenquex is this ticket relevant at all? If not it can be moved to won't fix

**Comment by phraenquex** [comment]
No, this shouldn't be relevant - sorry, meant to say in the call earlier.  

This is covered by #1071 .  

If you handle this error gracefully, then you can move this ticket to "invalid or duplicate".

**Comment by phraenquex** [comment]
Done.
